alma/controllers/hook/DisplayRefundsHookController.php

```diff
@@ -49,12 +49,12 @@ public function run($params)
     {
         $order = new Order($params['id_order']);
         if ($order->module !== 'alma') {
-            return;
+            return null;
         }
 
         $alma = ClientHelper::defaultInstance();
         if (!$alma) {
-            return false;
+            return null;
```

Comment:
Please add dockblock with :

* description
* `@return` and type + description (something like `null|string rendered template output`)
* and `@param array $params` + witch kind of `$params` we are waiting here